ViViaN/viv.conf

```diff
@@ -0,0 +1,11 @@
+<VirtualHost *:80>
+  ServerName localhost
+  ErrorLog /var/log/httpd/viv.err
+  CustomLog /var/log/httpd/viv.log combined
+  DocumentRoot /var/www/html/vivian
+  DirectoryIndex index.php
+  <Directory "/var/www/html/vivian">
```

Comment:
it may be nice to put the instance name after the html/ to allow for multiple instances (which the scripts already support)

Reply:
I will create a separate pull request making this file dynamic based on the provided instance name as well as update where the doc files are linked to.